packages/bpk-component-autosuggest/src/BpkAutosuggestSuggestion.tsx

```diff
@@ -16,31 +16,34 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
-import PropTypes from 'prop-types';
-import type { Node } from 'react';
+import type { ReactNode, FunctionComponent, SVGProps } from 'react';
 
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import { cssModules } from '../../bpk-react-utils';
 
 import STYLES from './BpkAutosuggest.module.scss';
 
 const getClassName = cssModules(STYLES);
 
 type Props = {
```

Comment:
I think we'd need to add support for `rest` to this type or extend the section html element interface (if there is one)?